### ISSUE TITLE
OAuthAccount.expires_at should be BigInteger

### ISSUE THREAD

**Issue by holtgrewe** [fastapi-users/fastapi-users#1296]
## Describe the bug

The `OAuthAccount.expires_at` field is an `Integer`. If your OAuth2 provider decides to give you a value in 2043 (as ORCID sandbox does) then this cannot be inserted into the database any more.

## To Reproduce

1. Setup fastapi-users with OAuth2.
2. Authenticate with a provider who returns a sufficiently large `expires_at` value.
3. Database insert will fail.

## Expected behavior

Dates in 2043 should work.

## Configuration
 - Python version : 3.11
 - FastAPI version : 0.103.1
 - FastAPI Users version : 12.1.2

### FastAPI Users configuration

N/A

## Additional context

N/A

**Comment by frankie567** [comment]
Good catch, indeed! PR welcome on [`fastapi-users-db-sqlalchemy`](https://github.com/fastapi-users/fastapi-users-db-sqlalchemy) :)

**Comment by UmukoroG** [comment]
@frankie567 may i please be assigned this bug?

**Comment by mo-hossam-stack** [comment]
Is any one working on this issue?
